uber/api.py

```diff
@@ -356,10 +356,10 @@ def login(self, first_name, last_name, email, zip_code):
         """
         #this code largely copied from above with different fields
         with Session() as session:
-            attendee_query = session.query(Attendee).filter_by(first_name=first_name,
-                                                               last_name=last_name,
-                                                               email=email,
-                                                               zip_code=zip_code)
+            attendee_query = session.query(Attendee).filter(Attendee.first_name.ilike(first_name),
+                                                               Attendee.last_name.ilike(last_name),
+                                                               Attendee.email.ilike(email),
+                                                               Attendee.zip_code.ilike(zip_code))
```

Comment:
Are you still trying to do exact name matches?  Or just matches which contain the name?  If you say

```python
Attendee.first_name.ilike(first_name)
```

then it will match ``will`` to ``Will`` but if you want to match ``will`` to ``William`` then you need to use wildcard characters like this:

```python
Attendee.first_name.ilike('%' + first_name + '%')
```